docs/how-to-use/search/advanced-search-user-docs.md

```diff
@@ -0,0 +1,65 @@
+# Advanced Search Component
+
+## Introduction
+
+The `AdvancedSearchComponent` allows for the creation of complex queries using a graphical widget.
+The widget's contents are then turned into a string representing a Gravsearch (SPAQRL) query to be sent to DSP-API.
+
+A query consists of the following elements:
+- ontology (data model) selection
+- selection of a resource class belonging to the selected ontology (optional)
+- specification of properties, comparison operators, and values (optional).
+
+Although selection of a resource or a property ore both are optional, either a resource class has to be selected or at least one property has to be specified,
+otherwise the query is not considered valid and cannot be submitted.
+
+## Comparison Operators
+
+Depending on the value type of the chosen property,
+one or more of the following comparison operators can be selected:
+
+- `is equal to`: value equality: same number, exact same string, overlap of date periods, same target resource.
+- `is not equal to`: value inequality: not same number, not exact same string, no overlap of date periods, not same target resource.
+- `is greater than`: value comparison: number is greater than search value, date period begins after search value.
+- `is greater than or equal to` value equality / value comparison: number is equal or greater than search value, overlap of date periods or date period begins after search value.
+- `is less than`: value comparison: number is less than search value, date period ends before search value.
+- `is less than or equal to`: value equality / value comparison: number is equal or less than search value, overlap of date periods or date period ends before search value.
```

Comment:
equal **to**

Reply:
fixed in https://github.com/dasch-swiss/dsp-ui-lib/pull/295/commits/8d3c3f20a65273bfdbca05eb12bdd410426edd19